src/components/MCQ/MCQQuestion.vue

```diff
@@ -125,8 +126,14 @@ function getClassForTimedMode(_id: { $oid: string }, key: string): string {
     questionIndex,
     selectedValue,
   );
-  return String(answer) === key ? "selected" : "";
+  const answerString = String(answer);
```

Comment:
This is only being used once, so we can just do:

```ts
if (String(answer) === key){...}

Reply:
fixed